auditwheel/lddtree.py

```diff
@@ -37,7 +36,7 @@ def normpath(path: str) -> str:
     return os.path.normpath(path).replace('//', '/')
 
 
-def readlink(path: str, root: str, prefixed: bool=False) -> str:
+def readlink(path: str, root: str, prefixed: bool = False) -> str:
```

Comment:
I was trying to figure out where this change came from and realized this error is from a relatively new version of PEP8: https://github.com/python/peps/commit/acda0d1ddef5bc19c1f9410895d21555a9395726

Guess we must comply :woman_shrugging: